src/test/java/io/confluent/kafka/connect/datagen/DatagenTaskTest.java

```diff
@@ -318,19 +347,15 @@ public OffsetStorageReader offsetStorageReader() {
     task.start(config);
   }
 
+  private void loadKeyAndValueSchemasFromString(String schemaString, String keyFieldName) {
+    org.apache.avro.Schema avroSchema =
+        new Generator.Builder().schemaString(schemaString).build().schema();
```

Comment:
Doesn't look like we need to construct an entire `Generator` here just to parse a schema:

```suggestion
    org.apache.avro.Schema avroSchema =
        new org.apache.avro.Schema.Parser().parse(schemaString);
```